test-fixtures/slconfig-invalid-format.toml

```diff
@@ -0,0 +1,22 @@
+[server]
+  debug_trace = 123
+
+[bluemix]
+  iam_url = 123
+  iam_client_id = 123
+  iam_client_secret = 123
+  iam_api_key = 123
+  refresh_token = 123
+  containers_api_route = "api_endpoint"
+  containers_api_route_private = "private_api_endpoint"
```

Comment:
we will not need api route for vpc file, please clean up the code and make sure we will use iam private endpoint as default else whatever iam endpoint for token exchanges that we will use